lib/plugins/crowdsec/template.lua

```diff
@@ -7,7 +9,79 @@ function template.compile(template_str, args)
         template_str = template_str:gsub(var, v)
     end
 
-    return template_str
+    return findIfStatements(template_str, args)
```

Comment:
just curious, why is this needed ?

Reply:
Instead of handling multiple ban templates, I just created a quick conditional rendering function

https://github.com/crowdsecurity/lua-cs-bouncer/pull/45/files#diff-e2d2e7737a7c5b6687567dbc295347987cca1a5362ca92fa99dd5b440b7dc2a7R20

it find the if statements executes the conditional and if true or false it renders the component

Reply:
my bad, forgotten there actually no "proper" template engine